domains/client/domain-operator/src/aux_schema.rs

```diff
@@ -35,12 +40,9 @@ const LATEST_CONSENSUS_HASH: &[u8] = b"latest_consensus_hash";
 /// - Hash(ConsensusBlock12) => Hash(DomainBlock5)
 const BEST_DOMAIN_HASH: &[u8] = b"best_domain_hash";
 
-/// Prune the execution receipts when they reach this number.
-///
-/// NOTE: `PRUNING_DEPTH` must larger than the consensus chain `ConfirmationDepthK` to avoid
-/// accidentally delete non-confirmed ER from consensus forks that can potentially become the
-/// best fork in the future
-const PRUNING_DEPTH: BlockNumber = 1000;
+/// Tracks domain block hash and consensus block hash from which domain block is derived from
+/// at a given domain block height.
```

Comment:
Sorry to be a pain about this, but I couldn't work out how to split up the parts of that sentence when I read it

```suggestion
/// Tracks a domain block hash, and the consensus block hash it is derived from,
/// at a given domain block height.
```